### ISSUE TITLE
BigQuery exporter does not export ProgramArguments in Plist results

### ISSUE THREAD

**Issue by anttitikkanen** [google/grr#949]
When using the BigQuery exporter to export hunt results that contain MacOs plist entries, the "ProgramArguments" field will be missing (https://github.com/google/grr/blob/8ad8a4d2c5a93c92729206b7771af19d92d4f915/grr/proto/grr_response_proto/sysinfo.proto#L768). 

I suppose this is due to not having a special converter for Plists, so the default converted ignores repeated fields? (https://github.com/google/grr/blob/3adbc6b9d5a25766c30db49ce6ab7914c0f2bc63/grr/server/grr_response_server/export.py#L320)

**Comment by mbushkov** [comment]
Correct, an exporter for LaunchdPlist type has to be implemented. To do that, one would need to do the following:
* Define an `ExportedLaunchdPlist` protobuf and a corresponding Python class. The proto must contain the field `metadata` of type `ExportedMetadata` and shouldn't contain repeated fields (it can contain nested fields, though). 
* Implement the exporter that would convert `LaunchdPlist` values into `ExportedLaunchdPlist`.
